sabelotodo/routes.py

```diff
@@ -1,25 +1,54 @@
-from flask import abort, jsonify, current_app as app
-from .models import Item
+from flask import abort, jsonify, request, Response, current_app as app
+from marshmallow import ValidationError
+from .models import Item, ItemSchema, db
 from dataclasses import asdict
 
+item_schema = ItemSchema()
 
 @app.route('/')
 def hello():
     return "Hello World!"
 
 
-@app.route('/item')
+@app.route('/item', methods=["GET"])
 def all_items():
     return jsonify([asdict(i) for i in Item.query.all()])
 
 
-@app.route('/item/<itemid>')
-def item_by_id(itemid):
-    if itemid.isnumeric():
-        return_value = Item.query.get(itemid)
-        if return_value:
-            return asdict(return_value)
-    abort(404)
+@app.route('/item/<int:itemid>')
+def get_item_by_id(itemid):
+    item = Item.query.get_or_404(itemid)
+    return asdict(item)
+
+
+@app.route('/item/<int:itemid>', methods=["DELETE"])
+def delete_item_by_id(itemid: str):
+    item = Item.query.get_or_404(itemid)
+    db.session.delete(item)
+    db.session.commit()
+    return Response(status=200)
+
+
+@app.route('/item', methods=["POST"])
+def create_item():
+    json_data = request.get_json()
+    if not json_data:
+        return Response(status=400)
+
+    try:
+        item = item_schema.load(json_data)
+    except ValidationError:
+        return Response(status=400)
+
+    try:
+        db.session.add(item)
+        db.session.commit()
+    except:
```

Comment:
In general, it is best practices not to do a blanket except. I think try to figure out what kinds of errors might occur here and do an except for them specifically.